netbox/utilities/querydict.py

```diff
@@ -55,7 +55,7 @@ def prepare_cloned_fields(instance):
     for key, value in attrs.items():
         if type(value) in (list, tuple):
             params.extend([(key, v) for v in value])
-        elif value not in (False, None):
+        elif value not in (False, None) or value == 0:
```

Comment:
The second condition has no effect; `0` is neither False nor None.

Reply:
0 is False. I tested this code and it fixes the issue for me.

Reply:
It also breaks boolean fields, such as `enforce_unique` on the VRF model, which is why we specifically test for False.